assets/javascripts/discourse/components/group-reports-nav-item.js.es6

```diff
@@ -0,0 +1,23 @@
+import { ajax } from "discourse/lib/ajax";
+
+export default Ember.Component.extend({
+  group: null,
+  showReportsTab: false,
+
+  checkForReports() {
+    const p1 = ajax(`/g/${this.group.name}/reports`);
+    return p1.then(response => {
+      return this.set("showReportsTab", response.queries.length > 0);
+    });
+  },
+
+  init(args) {
+    this.set("group", args.group);
+    let usersGroupIds = this.currentUser.groups.map(g => g.id);
```

Comment:
Rather than mapping through then running an `includes`, you could do it in one go with something like: `if (this.currentUser.groups.some(g => g.id === this.group.id)`

Reply:
Agreed!